### PR TITLE
Fix framework edit styling

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Fix styling for analytical framework edit tabs

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/views/ProjectEdit/Framework/index.tsx [hunk 1]
```diff
@@ -225,13 +224,13 @@ function ProjectFramework(props: Props) {
 
     const handleNewFrameworkAddSuccess = useCallback((newFrameworkId: number) => {
         setSelectedFramework(newFrameworkId);
-        retriggerGetFrameworkListRequest();
+        setFrameworkList([]);
+        setOffset(0);
         hideFrameworkAddModal();
-    }, [setSelectedFramework, hideFrameworkAddModal, retriggerGetFrameworkListRequest]);
+    }, [setSelectedFramework, hideFrameworkAddModal]);
```
Comment (tnagorra) [inline app/views/ProjectEdit/Framework/index.tsx]: Why not retrigger?

File: app/views/AnalyticalFramework/FrameworkForm/index.tsx [hunk 2]
```diff
@@ -281,6 +287,14 @@ function FrameworkForm(props: FrameworkFormProps) {
                     return true;
                 }}
             />
+            <SubNavbarIcons>
+                <div className={styles.appBrand}>
+                    <Svg
+                        className={styles.logo}
+                        src={deepLogo}
+                    />
+                </div>
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/FrameworkForm/index.tsx:296]: Do we need to add this div?

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetList/index.tsx [hunk 3]
```diff
@@ -207,69 +207,71 @@ function WidgetList(props: Props) {
...
-                    disabled={disabled || widgetsDisabled}
-                />
-                <AddItem
-                    name="MULTISELECTWIDGET"
-                    // FIXME: use strings
-                    label="Multi Select"
-                    onAddClick={handleAddClick}
-                    disabled={disabled || widgetsDisabled}
-                />
+                <div className={styles.moreWidgetsList}>
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetList/index.tsx:210]: Do we need to add this div? but why?

File: app/views/AnalyticalFramework/FrameworkForm/Review/styles.css [hunk 4]
```diff
@@ -1,39 +1,28 @@
...
-            flex-grow: 1;
-            margin: calc(var(--dui-spacing-large) - var(--dui-spacing-extra-small)) var(--dui-spacing-extra-large);
-
-            .tabs {
-                .border-wrapper {
-                    display: flex;
-                    flex-grow: 1;
-                }
+    .card {
+        width: calc(100% - 240px - 10vw);
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/FrameworkForm/Review/styles.css:12]: Wut?

**Review by tnagorra — DISMISSED** [review]

File: app/views/AnalyticalFramework/FrameworkForm/Review/styles.css (hunk 4 above)
Comment (AdityaKhatri) [reply app/views/AnalyticalFramework/FrameworkForm/Review/styles.css:12]: yes, scientific calculation. This is the widht of side bar in framework edit.

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetList/index.tsx (hunk 3 above)
Comment (AdityaKhatri) [reply app/views/AnalyticalFramework/FrameworkForm/components/WidgetList/index.tsx:210]: Yes, expandable container's content cannot be set to flex, so need to create another div

File: app/views/AnalyticalFramework/FrameworkForm/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply app/views/AnalyticalFramework/FrameworkForm/index.tsx:296]: Yes according to Ankit

File: app/views/ProjectEdit/Framework/index.tsx (hunk 1 above)
Comment (AdityaKhatri) [reply app/views/ProjectEdit/Framework/index.tsx]: change in offset auto retriggers framework pull

Comment (tnagorra) [reply app/views/ProjectEdit/Framework/index.tsx]: omg, but what if offset is already 0?

Comment (AdityaKhatri) [reply app/views/ProjectEdit/Framework/index.tsx]: good point :P

**Review by samshara — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**